tests/TestKernel.php

```diff
@@ -48,7 +49,11 @@ public function registerContainerConfiguration(LoaderInterface $loader): void
                 'messenger' => [
                     'default_bus' => 'bus1',
                     'buses' => [
-                        'bus1' => [],
+                        'bus1' => [
+                            'middleware' => [
+                                AmqpFlushMiddlware::class => [],
```

Comment:
This is also annoying - the user would need to manually add this - though we could add this with `prepend` int he bundle but we then cannot guarantee it's going to be in the right position...